client/mgcp/driver/src/main/java/org/restcomm/media/client/mgcp/stack/JainMgcpStackProviderImpl.java

```diff
@@ -141,7 +142,7 @@ public void removeJainMgcpListener(JainMgcpListener listener) {
 		}
 	}
 
-	public void sendMgcpEvents(JainMgcpEvent[] events) throws IllegalArgumentException {
+	public void sendMgcpEvents(JainMgcpEvent[] events, InetAddress address, int port) throws IllegalArgumentException {
```

Comment:
Duplicate code. Can't `sendMgcpEvents(JainMgcpEvent[] events)` invoke `sendMgcpEvents(JainMgcpEvent[] events, null, -1)` ?

Reply:
Unfortunately not, i have kept that function for backward compatibility , since old code had only a function with single param

Reply:
Hi , i have created the tests , however they are part of LB testsuite. In order to bring them as part of MMS i will have to duplicate them

Reply:
but @knosach both methods seem to be identical... only one line changes, related to the input parameters. Why can't we have the following:

```
public void sendMgcpEvents(JainMgcpEvent[] events) throws IllegalArgumentException {
    sendMgcpEvents(events, null, -1);
}
```

> Hi , i have created the tests , however they are part of LB testsuite. In order to bring them as part of MMS i will have to duplicate them

OK, will wait for those.

Thanks